robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowLayoutLibrary.java.vm

```diff
@@ -0,0 +1,91 @@
+package org.robolectric.shadows;
+#if ($api == 21)
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.robolectric.util.ReflectionHelpers;
+import org.robolectric.util.ReflectionHelpers.ClassParameter;
+
+import java.io.File;
+import java.lang.reflect.Constructor;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+import android.renderscript.Sampler.Value;
+
+import com.android.ide.common.rendering.LayoutLibrary;
+import com.android.ide.common.rendering.api.Bridge;
+import com.android.ide.common.sdk.LoadStatus;
+import com.android.layoutlib.api.ILayoutBridge;
+import com.android.utils.ILogger;
+import com.android.utils.SdkUtils;
+
+/**
+ * Shadow for {@link com.android.ide.common.rendering.LayoutLibrary}.
+ */
+@Implements(value = LayoutLibrary.class, isInAndroidSdk = false)
+public class ShadowLayoutLibrary {
+  @Implementation
+  public static LayoutLibrary load(String layoutLibJarOsPath, ILogger log, String toolName) {
```

Comment:
This seems similar / same as the real implementation, what is the difference and do we need to shadow here?